airflow/contrib/hooks/gcp_dataproc_hook.py

```diff
@@ -208,13 +209,24 @@ def __init__(self, project_id, task_id, cluster_name, job_type, properties):
                 "placement": {
                     "clusterName": cluster_name
                 },
+                "labels": {'airflow-version': 'v' + version.replace('.', '-').replace('+', '-')},
```

Comment:
Nice!

Reply:
We have to add these improvements in a few other places also. It would be great to see what is doing GCP activities.